cves/kernel/CVE-2016-6156.yml

```diff
@@ -55,7 +55,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A race condition in the ec_device_ioctl_xcmd function caused a denial of
+  service, known as a "double fetch" vulnerability.
```

Comment:
It would be even better if there was an explanation of what a double fetch is and how it leads to a denial of service.